### PR TITLE
User job list

### PR DESCRIPTION
**Pull request by ExerciseAndrew**

### CODE REVIEW

File: src/app/pages/Jobs.jsx [hunk 1]
```diff
@@ -69,32 +69,39 @@ function Jobs() {
...
+          value={'Very Interested'}
+
         >
           Very Interested
         </Button>
         <Button
           variant="outline"
           style={Theme.palette.independence}
           onClick={selectJobList}
+          value={'Interested'}
```
Comment (maki-q) [inline src/app/pages/Jobs.jsx:102]: Strings can be passed in directly, ie. value="Interested"

File: src/app/pages/Jobs.jsx [hunk 2]
```diff
@@ -69,32 +69,39 @@ function Jobs() {
...
+          value={'Extremely Interested'}
+
         >
           Extremely Interested
         </Button>
         <Button
           variant="outline"
           style={Theme.palette.independence}
           onClick={selectJobList}
+          value={'Very Interested'}
```
Comment (maki-q) [inline src/app/pages/Jobs.jsx:93]: Strings can be passed in directly, ie. value="Interested"

File: src/app/pages/Jobs.jsx [hunk 3]
```diff
@@ -69,32 +69,39 @@ function Jobs() {
...
           onClick={selectJobList}
+          value={'Applied'}
         >
           Applied
         </Button>
         <Button
           variant="outline"
           style={Theme.palette.independence}
           onClick={selectJobList}
+          value={'Extremely Interested'}
```
Comment (maki-q) [inline src/app/pages/Jobs.jsx:84]: Strings can be passed in directly, ie. value="Interested"

File: src/app/pages/Jobs.jsx [hunk 4]
```diff
@@ -69,32 +69,39 @@ function Jobs() {
         <JobSearch />
       </Stack>
       <Box style={{ display: 'flex', flexDirection: 'row', justifyContent: 'center' }}>
-        <h1>My Jobs</h1>
+        <Typography>My Jobs</Typography>
         <Button
           variant="outline"
           style={Theme.palette.independence}
           onClick={selectJobList}
+          value={'Applied'}
```
Comment (maki-q) [inline src/app/pages/Jobs.jsx:76]: Strings can be passed in directly, ie. value="Interested"

**Review by maki-q — CHANGES_REQUESTED** [review]
@ExerciseAndrew Small changes to fix, TY

File: server/controllers/savedJobsController.js [hunk 5]
```diff
@@ -48,10 +48,7 @@ exports.saveJob = async (req, res) => {
 };
 
 exports.getSavedJobs = (req, res) => {
-// Kevin and Duncan will add the userID. in the meantime, use hard coded:
-  req.user = {
-    id: 1,
-  };
+  console.log('user id', req.user.id)
```
Comment (Gobleizer) [inline server/controllers/savedJobsController.js]: Please remove console log, and instead insert a statement here that will send a 401 back to the user if they are not logged in.  See the example in Slack.

File: src/app/components/UsersJobList/UserJobItem.jsx [hunk 6]
```diff
@@ -34,14 +21,17 @@ const UserJobItem = (props) => {
...
+        </Typography>
         <Typography variant="body2">
           {props.job.description}
           <br />
           {props.job.date}
         </Typography>
       </CardContent>
       <CardActions>
-        <Button size="small" onClick={(e) => { jobSelect(props.job) }}>Learn More</Button>
+        <Button size="small" onClick={(e) => {console.log('e', e)}}>Apply</Button>
```
Comment (Gobleizer) [inline src/app/components/UsersJobList/UserJobItem.jsx]: Please remove console log if unneeded for further development work

**Review by Gobleizer — CHANGES_REQUESTED** [review]
Please see comments for requested changes.

**Pull request closed without merging**